From 019263cca062ce34c826102b7e1191a26947baa4 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Wed, 29 Apr 2015 14:21:01 +0800
Subject: [PATCH] MLK-10773-1 ARM: dts: imx6ul: config proper clocks for pxp

Config pxp's clocks property in dts file for 6ul
platform.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index d677af5..6b240f6 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -839,7 +839,7 @@
 				compatible = "fsl,imx6ul-pxp-dma", "fsl,imx6sl-pxp-dma", "fsl,imx6dl-pxp-dma";
 				reg = <0x021cc000 0x4000>;
 				interrupts = <GIC_SPI 8 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX6UL_CLK_DUMMY>,
+				clocks = <&clks IMX6UL_CLK_PXP>,
 					 <&clks IMX6UL_CLK_DUMMY>;
 				clock-names = "pxp-axi", "disp-axi";
 				power-domains = <&gpc 2>;
-- 
1.8.0

